machine-learning/sentiment-analysis/README.md

```diff
@@ -0,0 +1,34 @@
+# Sentiment classification with TensorFlow
+
+***This is an experimental workflow. It doesn't work on production environment yet***
+
+This example introduces sentiment analysis for movie reviews using TensorFlow and TensorFlow Hub.
+Original example is written in [the official document](https://www.tensorflow.org/hub/tutorials/text_classification_with_tf_hub).
+
+This workflow will:
+1. Fetch review data from Treasure Data
+2. Build a model with TensorFlow
+3. Store the model on S3
+4. Predict polarities for unknown review data and write back to Treasure Data
+
+Currently, prediction with fetching model from S3 is not evaluated yet.
```

Comment:
Is this non-working example should be published now? Or, better to remain WIP for our customers?

Reply:
`sentiment.py` does 1) training sentiment predictive model with pre-trained model 2) predict with the model and store prediction results 3) store trained model to S3, without OOM. I think it is minimal code for showing capability with working on TF.

`predict.py` is optional scenario that allows prediction with trained model trained in `sentiment.py` and it still worth to put it on the repo. The memory limitation should be a product related issue and it could be changed in future. But I believe it is a realistic use case and should be worth to put in the repo. (I'm not intend to write explicitly in the doc though)

Reply:
So, LGTM to merge.